doc/source/whatsnew/v1.3.0.rst

```diff
@@ -971,6 +971,7 @@ Other
 - Bug in :func:`pandas.util.show_versions` where console JSON output was not proper JSON (:issue:`39701`)
 - Bug in :meth:`DataFrame.convert_dtypes` incorrectly raised ValueError when called on an empty DataFrame (:issue:`40393`)
 - Bug in :meth:`DataFrame.clip` not interpreting missing values as no threshold (:issue:`40420`)
+- Bug in :meth:`_CustomBusinessMonth.apply` not applying extra offset when initially rolled date `new` is already a business day (:issue:`41356`)
```

Comment:
_CustomBusinessMonth is not user-facing.  can this be phrased in terms of something users will directly see?

Reply:
Thanks for the feedback! I just re-phased the entry. Is it more along the line of what you were thinking?